README.md

```diff
@@ -2,26 +2,15 @@
 
 Yet Another Interpreter Written In Rust
 
-
 ## CI
 
 [Buildbot](https://ci.soft-dev.org/#/builders/1)
 
 ## Tests
 
 ```shell
-# run test locally
-$ make docker-test
-
-# run test in docker container
-$ make docker-test
-```
-
-## Docker
-
-```shell
-# docker build
-docker build -f .buildbot_dockerfile_debian . -t yaiwr-local --build-arg CI_UID=666
-# docker run
-docker run -t yaiwr-local:latest
+# run unit tests
+$ cargo test
+# run test in a container
+$ run_docker_ci_job # optional (--prune)
```

Comment:
I think it's just `-p` not `--prune`.

Reply:
I think its both: https://github.com/softdevteam/buildbot_config/blob/master/bin/run_docker_ci_job#L88
when I tested locally I used `--prune` but either should work

Reply:
Ah, I thought I'd remove the long form. Guess not. Oh well :)